src/app/containers/MediaPlayer/index.test.jsx

```diff
@@ -1,106 +1,32 @@
-import React from 'react';
-import { string, shape, node } from 'prop-types';
 import { shouldMatchSnapshot, isNull } from '@bbc/psammead-test-helpers';
-import MediaPlayerContainer from '.';
-import { RequestContextProvider } from '#contexts/RequestContext';
-import { ServiceContextProvider } from '#contexts/ServiceContext';
-import { ToggleContext } from '#contexts/ToggleContext';
-import { validVideoFixture, missingVpidFixture } from './helpers/fixtures';
-
-const defaultToggles = {
-  local: {
-    mediaPlayer: {
-      enabled: true,
-    },
-  },
-  test: {
-    mediaPlayer: {
-      enabled: true,
-    },
-  },
-  live: {
-    mediaPlayer: {
-      enabled: false,
-    },
-  },
-};
-
-const mockToggleDispatch = jest.fn();
-const ContextWrapper = ({ platform, children, toggleState }) => (
-  <RequestContextProvider
-    isAmp={platform === 'amp'}
-    service="news"
-    statusCode={200}
-    platform={platform}
-    id="foo"
-    pageType="article"
-    pathname="/pathname"
-  >
-    <ServiceContextProvider service="news">
-      <ToggleContext.Provider
-        value={{ toggleState, toggleDispatch: mockToggleDispatch }}
-      >
-        {children}
-      </ToggleContext.Provider>
-    </ServiceContextProvider>
-  </RequestContextProvider>
-);
-
-ContextWrapper.propTypes = {
-  children: node.isRequired,
-  platform: string.isRequired,
-  toggleState: shape({}),
-};
-
-ContextWrapper.defaultProps = {
-  toggleState: defaultToggles,
-};
+import {
+  VideoFixture,
+  VideoAmpFixture,
+  VideoFixtureNoPlaceHolder,
+  VideoFixtureNoVersionId,
+  VideoFixtureToggledOff,
+} from './fixtureData';
 
 describe('MediaPlayer', () => {
   describe('is called correctly', () => {
```

Comment:
I think this describe block is a bit redundant, it doesn't add much descriptive value.

Reply:
Yep I agree, this was just the naming of it before the refactor. How about changing it to `is rendered` and then each snapshot test will mention what the media player is rendered with or do you suggest completely removing this describe block?